build_runner/lib/src/build_script_generate/build_script_generate.dart

```diff
@@ -52,18 +52,28 @@ Future<Iterable<Expression>> _findBuilderApplications(String configKey) async {
         package.name, package.dependencies.map((n) => n.name), package.path);
   }
 
-  final builderDefinitions =
-      (await Future.wait(orderedPackages.map(_packageBuildConfig)))
-          .expand((c) => c.builderDefinitions.values)
-          .where((definition) {
+  bool _isValidDefinition(dynamic definition) {
```

Comment:
as a followup we should see whether there is some Interface that would make sense for this. Maybe something like `PackageProvidedCode` that has a `package` and `import` field. Then make `BuilderDefinition implement PackageProvidedCode`.

Reply:
acknowledged